workspace/apps/pidp/src/app/features/shell/pages/support-error/support-error.page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+
+import { provideAutoSpy } from 'jest-auto-spies';
+
+import { SupportErrorPage } from './support-error.page';
+
+describe('SupportErrorPage', () => {
+  let component: SupportErrorPage;
+  let router: Router;
+
+  const mockActivatedRoute = {
+    shellRoutes: {
+      MODULE_PATH: '',
+    },
+  };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [SupportErrorPage, provideAutoSpy(Router)],
+    });
+    router = TestBed.inject(Router);
+    component = TestBed.inject(SupportErrorPage);
+  });
+
+  describe('METHOD: onBack', () => {
+    given('user wants to navigate back to the PIDP', () => {
```

Comment:
I wouldn't mention the app name in the test since if we lifted this out into a lib would it be used by the same app?  maybe but maybe not - `user wants to be redirected back to the application`